mediathread/templates/projects/sequence.html

```diff
@@ -151,6 +151,17 @@ <h5 class="modal-title">Save Changes</h5>
                                             </label>
                                         </li>
                                         {% endif %}
+                                        <li>
```

Comment:
This looks good. One minor fix - Can you move the "Instructor - only author(s) and instructor can view" option after the "Draft - only you can view"? That's the order we use in other assignments.

Reply:
Got it!